model/prelude.sail

```diff
@@ -329,3 +332,26 @@ val def_spc_backwards : string -> unit
 function def_spc_backwards s = ()
 val def_spc_matches_prefix : string -> option((unit, nat))
 function def_spc_matches_prefix s = opt_spc_matches_prefix(s)
+
+val "print_int" : (string, int) -> unit
```

Comment:
I think this might be in the main Sail library?

Reply:
I'll remove this one.